must-call-qual/src/main/java/org/checkerframework/checker/mustcall/qual/MustCall.java

```diff
@@ -0,0 +1,40 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.DefaultFor;
+import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/**
+ * If an expression has type {@code @MustCall({"m1", "m2"})}, then its value might have an
+ * obligation to eventually call methods {@code m1}, {@code m2}, both, or neither before any exit
+ * point or de-allocation. The value cannot have an obligation to call any methods that are not
+ * named in the annotation.
+ *
+ * <p>The subtyping relationship is:
+ *
+ * <pre>{@code @MustCall({"m1"}) <: @MustCall({"m1", "m2"})}</pre>
+ *
+ * <p>For example, an object with an obligation to call only "m1" can be passed to a method whose
```

Comment:
"an obligation to call only m1" does not refer to anything that this type system can represent.  Should "only" be "at most"?  Likewise on the next line, should "both" be "at most both"?

Reply:
It is awkward to say "at most" every time one mentions "obj must call foo". I've added a note as part of the new documentation in the README.md file that should make it clear that any uses of the shorthand "obj must call foo" actually mean "obj must call at most foo".